modules/material/themes/material/core/loginuserpass.php

```diff
@@ -82,16 +83,18 @@ function onRecaptchaLoad() {
                         <?php
                         $username = htmlentities($this->data['username'] ?? null);
                         ?>
-                        <input type="text" required name="username" class="mdl-textfield__input mdl-color-text--black" value="<?= $username ?>"
-                               <?= empty($username) ? 'autofocus' : '' ?> id="username"/>
+                        <input type="text" required name="username" class="mdl-textfield__input mdl-color-text--black"
+                               value="<?= $username ?>"
+                            <?= empty($username) ? 'autofocus' : '' ?> id="username"/>
                     </div>
 
                     <div class="mdl-textfield mdl-js-textfield mdl-textfield--floating-label">
                         <label for="password" class="mdl-textfield__label">
                             <?= $this->t('{material:login:label_password}') ?>
                         </label>
 
-                        <input type="password" required name="password" class="mdl-textfield__input mdl-color-text--black" <?= ! empty($username) ? 'autofocus' : '' ?>
+                        <input type="password" required name="password"
+                               class="mdl-textfield__input mdl-color-text--black" <?= !empty($username) ? 'autofocus' : '' ?>
```

Comment:
Ugly line wrap could have been avoided by putting the <?= portion on the next line.